mustafaalispk/MyRecipe
# Add admin recipe and admin recipe details page

Add ability to list recipes pending approval including details page

# How to test

1. Create new recipe.
2. Open recipes admin is page, verify recipe displayed in list.
3. View details pending recipe.
